docs/config-presets-and-options.md

```diff
@@ -0,0 +1,56 @@
+# Configuration Presets and Options
+
+## Defining the Cluster Topology
+
+Openshift clusters can have different topologies. In Openshift development we refer to 
+these different topologies as different installations with adjectives and acronyms. 
+Below is the list of cluster toplogies currently being used to test the installers:
+
+|                       | Master Nodes | Worker Nodes | Masters <br>Schedulable |
+|-----------------------|--------------|--------------|-------------------------|
+| High Availablity (HA) |       3      |       2      |           No            |
+| Compact               |       3      |       0      |           Yes           |
+| Single Node Openshift |       1      |       0      |           Yes           |
+
+### High Availability (HA)
+
+|                     | Production | Development |
+|---------------------|------------|-------------|
```

Comment:
I'm not sure about the `Production` column meaning here since `dev-scripts` it's just for development.

Reply:
I wasn't sure what to word this set of configurations. This is the configuration listed as a the minimum requirements for the nodes in the OpenShift documentation itself. Development is the dev-scripts defaults, which don't work with Assisted Installer out of the box without an override.

Reply:
It's more of a "documented minimum requirement"

Reply:
Here is the link to the documentation: https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal/installing-bare-metal.html#installation-minimum-resource-requirements_installing-bare-metal

Reply:
I updated the format on this to be more descriptive